build-configs/obdach/assets/intro-slides/Events.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Delete all intro slides and native assets

Reply:
alright @steffenkleinle i removed unneeded assets